src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -89,53 +89,144 @@
  * <pre>
  * &lt;module name=&quot;RequireThis&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
```

Comment:
please update this code block to have more violations to show how violations disappear one used put "false" in config.

Reply:
Same reasoning from comment above.